Philips.CodeAnalysis.AnalyzerPerformance/Program.cs

```diff
@@ -1,6 +1,5 @@
 ﻿// © 2023 Koninklijke Philips N.V. See License.md in the project root for license information.
 
-using Microsoft.Build.Logging.StructuredLogger;
 using Philips.CodeAnalysis.Common;
 
```

Comment:
Remove this file from the PR. The change serves no purpose, yet causes the build to fail.